justfile

```diff
@@ -79,20 +79,20 @@ docs-serve params="":
 [doc("Ruff format")]
 [group("linter")]
 ruff-format *params:
-  uv run --active ruff format {{params}}
+  uv run --active --frozen ruff format {{params}}
```

Comment:
`--frozen` required here to prevent unintentional modification of `uv.lock` after `uv run` was called from pre-commit local hooks, otherwise each call to `just lint` will change `uv.lock`

I think it's better to prevent editing of `uv.lock`, but if you have other thoughts, please let me know